priv/templates/authority_ecto.gen.context/lock/reason.ex

```diff
@@ -0,0 +1,19 @@
+defmodule <%= inspect context.lock_reason.module %> do
```

Comment:
I think I would like to avoid generating a file for `Accounts.Token.Purpose` and `Accounts.Lock.Reason`.

I have four potential ideas for how we might do that:
1. Ship `Authority.Ecto` with these custom types pre-defined. (e.g. `Authority.Ecto.Purpose`).
2. `Authority.Ecto` can provide yet another macro to generate an Ecto enum. (e.g. `Authority.Ecto.Enum`)
3. Tell users to install an enum library and generate code under that assumption. (e.g. exnumerator)
4. Just use strings instead of atoms in `Authority.Ecto.Template` when generating changesets. Then, use `validate_inclusion` in the changeset.